test/test_serialization.py

```diff
@@ -308,11 +307,12 @@ def __reduce_ex__(self, proto):
                             self.tensor.size())))
 
         with tempfile.NamedTemporaryFile() as f:
-            torch.save({"spoofed": TensorSerializationSpoofer(x)}, f.name)
+            torch.save({"spoofed": TensorSerializationSpoofer(x)}, f)
```

Comment:
This changes semantic a bit, if we want to enable it on Windows, shouldn't we just use good old `mkstmp`?